interproscan/5.72-103.0/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM mambaorg/micromamba:2.0.5
+
+ARG VERSION=5.72-103.0
```

Comment:
out of curiosity, does this do anything different to 
`LABEL software.version="5.72-103.0"`
?

Reply:
VERSION is an env variable that I use in the rest of the dockerfile